File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    // 1
    // var answer = '';
    // for (let i of rsp) {
    //     if (i == '0') {
    //         answer += '5'
    //     }
    //     else if (i == '2') {
    //         answer += '0'
    //     }
    //     else if (i == '5') {
    //         answer += '2'
    //     }
    // }
    // return answer;
    
    // 2
    let answer = {'0':'5', '2':'0', '5':'2'};
    return [...rsp].map(v => answer[v]).join('')
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.py
def solution(rsp):
    # 1
    # answer = ''
    # for i in rsp :
    #     if i == '0' :
    #         answer += '5'
    #     elif i == '2' :
    #         answer += '0'
    #     elif i == '5' :
    #         answer += '2'
    # return answer
    
    # 2
    d = {'0':'5', '2':'0', '5':'2'}
    return ''.join([d[i] for i in rsp])